### ISSUE TITLE
Decide on scientific questions

### ISSUE THREAD

**Issue by ha0ye** [weecology/MATSS-LDATS#18]
(review MATSS-y grants from 2018 for fleshed out questions)

Some basic ideas to start:
* distribution of the # of topics in communities?
* are discrete changepoints or trends more common?
* how many changepoints? what is the modal length of time between changepoints?
* "patterns" of results associated with richness, abundance, sampling frequency, TS length, taxa, biome
* tests of indicators (or predictors) of changepoints / regime-shifts
* the relationship between system or time series properties and changepoint number/type

deadline: ~~February 28, 2019~~

**Comment by diazrenata** [comment]
From April 18 meeting notes https://github.com/weecology/MATSS/issues/114:
Empirical:
- What do we see:
        - how many communities does the LDA find
        - LDA scaling
        - species richness // k
        - spatial scale: downsampling
        - species richness, temporal beta diversity // k
           - core-transient? Ellen
           - species: are there transient species who show up as centers of LDA communities vs transients that don't drive signals in the LDA community because they are ephemeral
        - what kinds of dynamics do we see: nchangepoints, trends
- Where do we see it:
    - decompose into different ecosystems, taxa, etc
- Why do we see it:
    - data on potential drivers

Simulation based (Juniper):
- What can we get out of a dataset, depending on its characteristics
        - number of samples, number of species, number of time points
        - complexity of timeseries
        - downsampling and simulating
- Functionality for LDATS

**Comment by ha0ye** [comment]
> @ha0ye Thanks for sharing this; I'm interested in seeing what we gain/lose by doing the LDA step relative to a JSDM- or HGAM-like model; where the cross-over point is in terms of number of taxa in the series that some dimension reduction is needed, for example. Is the change-point model still the same thing that Dave Harris put together with Erica & Morgan or something different?

@gavinsimpson Yup, the method being used is being refined by @juniperlsimonis over in https://github.com/weecology/LDATS .

(I'm responding to your Q here, so that the folks involved with this project can keep track of the conversation)

**Comment by gavinsimpson** [comment]
Thanks @ha0ye. If it's OK with you I'll also comment back over in the other repo as my question was also motivated by a desire to group all the "regime shift" methods into some broader groups (the changepoint method in LDATS is a form of mean and/or variance shift model with k breakpoints, and many of the regime shift methods also fall into that category).
